test/images/crd-conversion-webhook/converter/framework.go

```diff
@@ -33,6 +33,10 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer/json"
 )
 
+// A debugging variable that incremeneted on each conversion call and will be added to the converted object.
+// Note that a real converter may not do this.
+var conversionIterationID = 0
```

Comment:
hrmm... is there any other way to verify this? conversion not being idempotent seems like it could cause a lot of problems (even in our tests)